plugin-openshift/che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/project/wizard/page/configure/ConfigureProjectPresenter.java

```diff
@@ -96,24 +95,34 @@ public void apply(List<ProjectDescriptor> projects) throws OperationException {
     }
 
 
-    /** {@inheritDoc} */
     @Override
     public boolean isCompleted() {
         if (view.isNewOpenShiftProjectSelected()) {
+            setUpNewProjectRequest();
+            setUpCheProjectRequest();
             return isOpenShiftProjectNameValid(view.getOpenShiftNewProjectName()) & isCheProjectNameValid(view.getCheNewProjectName());
         } else {
+            setUpExistProjectRequest();
+            setUpCheProjectRequest();
             return view.getExistedSelectedProject() != null && isCheProjectNameValid(view.getCheNewProjectName());
         }
     }
 
-    private boolean isOpenShiftProjectNameValid(String name) {
-        if (openShiftProjects.contains(name)) {
+    /**
+     * Checks whether project name matches OpenShift's naming requirements.
+     * If is not valid, display error message on the view, depending on case.
+     * @param projectName - project name
```

Comment:
This javadoc is not formatted.
Params should be separated from javadoc description with one empty line, also `@param` description should be written in a new line without dash `-`.

See [formatter configuration](https://github.com/codenvy/codenvy-dev-resources/blob/master/che-codestyle/src/main/resources/che-codestyle-intellij-new.xml).
See javadoc [example](https://github.com/codenvy/codenvy-dev-resources/wiki/Formatting#javadoc).